autopush/endpoint.py

```diff
@@ -336,7 +336,8 @@ def _invalid_auth(self, fail):
         if isinstance(fail.value, VapidAuthException):
             raise fail.value
         message = fail.value.message or repr(fail.value)
-        if isinstance(fail.value, AssertionError):
+        if (isinstance(fail.value, AssertionError) or
+                isinstance(fail.value, ValueError)):
```

Comment:
There are a number of reasons that either exception could be raised, that would be difficult to relay. The ValueError in the exception is due to an bad JWT key, for instance. The message is already pointing that the bearer token (and thus VAPID) is failing. It may be worth pointing them toward documentation, though.